content/source/docs/enterprise/private/install-installer.html.md

```diff
@@ -80,6 +85,24 @@ For Linux distributions other than RHEL, check Docker compatibility:
 1. If a firewall is configured on the instance, be sure that traffic can flow out of the `docker0` interface to the instance's primary address. For example, to do this with UFW run: `ufw allow in on docker0`. This rule can be added before the `docker0` interface exists, so it is best to do it now, before the Docker installation.
 1. _Optional_: Get a domain name for the instance.  If you opt to use only an IP to access the instance, enter the IP into the hostname field when prompted during the web portion of the setup.
 
+
+#### Trusting SSL/TLS Certificates
+
+The installer has a section that allows multiple certificates to be specified as trusted. These
+certificates are a `Certificate Authority (CA) Bundle` and are used to allow PTFE to connect
```

Comment:
"These certificates are a" → "… are contained in a"?  This parses strangely for me.